spa_ui/self_service/client/app/components/navigation/_navigation.sass

```diff
@@ -1,4 +1,7 @@
 .navbar-pf-alt .navbar-brand .navbar-brand-name
  height: 20px
  margin-top: 10px
+
+.nav-pf-vertical-alt .list-group-item.active
+ background: #e3f4fd
```

Comment:
try avoiding declaring one off colors, if possible add it to the library (_colors.sass) and then import into the app, (_app_colors.sass) and reference accordingly